examples/connectivity/v4/port2serviceprofilepublic/main.tf

```diff
@@ -37,7 +37,7 @@ resource "equinix_fabric_connection" "port2profile" {
       }
       link_protocol {
         type= var.aside_link_protocol_type
-        vlan_s_tag= var.aside_link_protocol_stag
+        vlan_tag= var.aside_link_protocol_stag
```

Comment:
Similar to other comments about variable matching context where it will be used.